### PR TITLE
Added LocationUtilsTest for LocationUtils

### PR DESCRIPTION
**Pull request by 4D17Y4**
**Description (required)**
Added LocationUtilsTest for LocationUtils

**Tests performed (required)**

Tested BetaDebug on Nokia 6.1+ API level 29.

### CODE REVIEW

File: app/src/test/kotlin/fr/free/nrw/commons/utils/LocationUtilsTest.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package fr.free.nrw.commons.utils
+
+import fr.free.nrw.commons.location.LatLng
+import org.hamcrest.MatcherAssert.assertThat
```
Comment (madhurgupta10) [inline app/src/test/kotlin/fr/free/nrw/commons/utils/LocationUtilsTest.kt:4]: Delete this unused import

File: app/src/test/kotlin/fr/free/nrw/commons/utils/LocationUtilsTest.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+/**
+ * Test class for location utils
+ */
+class LocationUtilsTest {
+
+    /**
+     * MapBox LatLng to commons LatLng test.
+     */
+    @Test
+    fun testMapBoxLatLngToCommonsLatLng() {
```
Comment (madhurgupta10) [inline app/src/test/kotlin/fr/free/nrw/commons/utils/LocationUtilsTest.kt:17]: Add a check for accuracy also!

File: app/src/test/kotlin/fr/free/nrw/commons/utils/LocationUtilsTest.kt [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        val commonsLatLngTest = LocationUtils.mapBoxLatLngToCommonsLatLng(com.mapbox.mapboxsdk.geometry.LatLng(0.0, 0.0))
+        assertEquals(0.0, commonsLatLngTest.latitude)
+        assertEquals(0.0, commonsLatLngTest.longitude)
+    }
+
+    /**
+     * Commons LatLng to MapBox LatLng test.
+     */
+    @Test
+    fun testCommonsLatLngToMapBoxLatLng() {
```
Comment (madhurgupta10) [inline app/src/test/kotlin/fr/free/nrw/commons/utils/LocationUtilsTest.kt:27]: Add a check for altitude also!

**Review by madhurgupta10 — CHANGES_REQUESTED** [review]

**Comment by 4D17Y4** [comment]
Done @madhurgupta10 .
Thanks.

**Review by madhurgupta10 — APPROVED** [review]

**Review by neslihanturan — APPROVED** [review]
Thanks @4D17Y4 :)

**Pull request merged**